Rover/Parameters.cpp

```diff
@@ -667,6 +667,13 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("GUID_OPTIONS", 52, ParametersG2, guided_options, 0),
 
+    // @Param: ROVER_OPTIONS
+    // @DisplayName: Rover mode options
+    // @Description: Rover mode specific options
+    // @Bitmask: 0:Enable steering speed scaling in MANUAL mode
+    // @User: Advanced
+    AP_GROUPINFO("ROVER_OPTIONS", 53, ParametersG2, rover_options, 0),
```

Comment:
I wonder if we can think of a better name for this parameter because it may lead to confusion as boat users wonder, "does this apply to boats as well?".  I guess it is equivalent to FLIGHT_OPTIONS in copter and plane..

It might be best to just name it MANUAL_OPTIONS and keep it specific to manual mode.